File: Bronze/5086/5086.cpp
// [5086] 배수와 약수
#include <iostream>
using namespace std;

int main(void)
{
    int a, b;
    while (1)
    {
        cin >> a >> b;
        if (!(a && b))
            return 0;
        
        // 배수라면?
        if (a % b == 0)
            cout << "multiple\n";
        
        // 약수라면?
        else if (b % a == 0)
            cout << "factor\n";
        
        else
            cout << "neither\n";
    }
}

File: Bronze/5086/5086.java
// [5086] 배수와 약수
import java.util.Scanner;

public class Main {
    public static void main(String[] args)
    {
        Scanner sc = new Scanner(System.in);
        int a, b;

        while (true) {
            a = sc.nextInt();
            b = sc.nextInt();

            if (a == 0 && b == 0)
                return;

            // 배수라면?
            if (a % b == 0)
                System.out.println("multiple");

            // 약수라면?
            else if (b % a == 0)
                System.out.println("factor");

            else
                System.out.println("neither");
        }
    }
}